app/src/snippets.js

```diff
@@ -0,0 +1,61 @@
+/* eslint no-template-curly-in-string: "off" */
+
+const snippets = {
```

Comment:
is it possible to externalize the snippets? like loading them via a json file which can be served by the backend?

it seems like people will want to add to the snippets and it would be a bummer to have to roll another release to change the snippets (but maybe that is another ticket?)

Reply:
totally, this is a great idea.  i'm not 100% sure how it would work but i'd like to pursue it.  maybe snippets could live in the data dir?

a nice side benefit is that folks could potentially modify ours if they had better ideas.

Reply:
following up on that: https://github.com/monome/maiden/issues/107